### DISCUSSION TITLE
"CapsLock Indicator" folder should be renamed to "CapsLockIndicator"

### DISCUSSION THREAD

**Discussion by KodeXall** [jonaskohl/CapsLockIndicator#128] [Ideas]
A user is unable to find the config file because a user is unlikely to assume that software's folder with such file will have a name spelled in a different way - thus those names such be matched precisely

This suggestion comes straight out of this incident: https://github.com/jonaskohl/CapsLockIndicator/discussions/123

**Comment by jonaskohl** [comment]
As I stated in #123, the folder is exactly named as the software itself (both spelled with one space) and is unlikely to change, as the folder name is publicly documented in [this knowledgebase article](https://cli.jonaskohl.de/kb/?a=settingsfile).
